src/Layout/Directory.Build.targets

```diff
@@ -14,7 +14,8 @@
     <FullNugetVersion Condition="'$(VersionSuffixDateStamp)' != '' and '$(VersionSuffixBuildOfTheDay)' != ''">$(FullNugetVersion).$(VersionSuffixDateStamp).$(VersionSuffixBuildOfTheDay)</FullNugetVersion>
 
     <PgoTerm Condition="'$(PgoInstrument)' == 'true'">-pgo</PgoTerm>
-    <ArtifactNameWithVersionSdk>dotnet-sdk-internal$(PgoTerm)-$(FullNugetVersion)-$(ProductMonikerRid)</ArtifactNameWithVersionSdk>
+    <!-- <ArtifactNameWithVersionSdk>dotnet-sdk-internal$(PgoTerm)-$(FullNugetVersion)-$(ProductMonikerRid)</ArtifactNameWithVersionSdk> -->
+    <ArtifactNameWithVersionSdk>dotnet-sdk-internal$(PgoTerm)-10.0.100-rc.1-$(ProductMonikerRid)</ArtifactNameWithVersionSdk>
```

Comment:
@DonnaChen888 Looks like this was created back when rc.1 was publicly available. With GA releasing tomorrow, let's update all of the spots we have rc.1 to point at the GA versions and try to get this merged later this week. 

It's passing all checks (other than known failures) so I think we're good to go later this week. I just didn't want it to go in with rc.1 which was a while ago now.